tests/test_cli_change_lb.py

```diff
@@ -169,6 +175,10 @@ def test_change_namespace_lb_group(caplog, two_gateways):
     assert f'"w_mbytes_per_second": "0",' in caplog.text
     assert f'"no_auto_visible": false,' in caplog.text
     assert f'"hosts": []' in caplog.text
+    try_change_one_namespace_lb_group_no_listeners(caplog, subsystem, "1", anagrpid2)
```

Comment:
Instead of "1" you can use anagrpid

Reply:
@gbregman , i do not understand theis comment. 1 is for nsid.